pom.xml

```diff
@@ -179,7 +179,7 @@
         <log4j.version>2.20.0</log4j.version>
         <mysql.jdbc.version>8.0.32</mysql.jdbc.version>
         <mockito.version>4.11.0</mockito.version>
-        <netty.version>4.1.100.Final</netty.version>
+        <netty.version>4.1.108.Final</netty.version>
```

Comment:
https://github.com/apache/kyuubi/pull/6520

Reply:
Seems gRPC is good with this Netty version